addon/components/frost-info-bar.js

```diff
@@ -5,5 +5,6 @@ export default Component.extend({
   // == Properties ============================================================
 
   classNames: ['frost-info-bar'],
-  layout
+  layout,
+  hook: 'info-bar'
```

Comment:
I'm kind of opposed to default hooks - this my be just my opinion, but I feel that it makes consumers "lazy" in that they never have to think about what a hook _should_ be called.  I think the usage of components/elements on a page should be a conscious activity, so the consumer kind of needs to say what that component _means_ to them in a given case.